File: Curso Logica de Programação Neri Neitzke/Algoritmos em Pascal/pascal/operacoes.pas
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
{Ler 2 números, efetuar as 4 operações matemáticas e mostrar os resultados.}
program operacoes;
    var num1, num2, soma, multiplicacao, divisao, subtracao : real;
begin

   //num1 := 10;
   //num2 := 5;

   write('Digite o numero 1 .: ');
   readln(num1);

   write('Digite o numero 2 .: ');
   readln(num2);


   soma := num1 + num2;
   multiplicacao := num1 * num2;
   divisao := num1 / num2;
   subtracao := num1 - num2;

   writeln('A soma = ',soma:6:2);  //saída
   writeln('A multiplicacao = ',multiplicacao:6:2);  //saída
   writeln('A divisao = ',divisao:6:2);  //saída
   writeln('A subtracao = ',subtracao:6:2);  //saída
   write('...................Final do Sistema..................... ');
   readln();
end.

File: Curso Logica de Programação Neri Neitzke/Algoritmos em Java/operacoes.java
// Prof Neri Aldoir Neitzke
// www.INFORMATICON.com.br - videoaulasneri@gmail.com
import javax.swing.JOptionPane;
class operacoes
{
   public static void main(String args[])
   {
      float num1, num2, soma, multiplicacao, subtracao, divisao;
      //num1=50;
      //num2=10;
      num1 = Float.parseFloat(JOptionPane.showInputDialog("Digite o numero 1"));
      num2 = Float.parseFloat(JOptionPane.showInputDialog("Digite o numero 2"));

      soma = num1 + num2;
      multiplicacao = num1 * num2;
      divisao = num1 / num2;
      subtracao = num1 - num2;
      //JOptionPane.showMessageDialog(null,"A soma = "+soma);
      //JOptionPane.showMessageDialog(null,"A divisao = "+divisao);
      //JOptionPane.showMessageDialog(null,"A multiplicacao = "+multiplicacao);
      //JOptionPane.showMessageDialog(null,"A subtracao = "+subtracao);

      JOptionPane.showMessageDialog(null,"A soma = "+soma+ "\nA divisao = "+divisao+
      "\nA multiplicacao = "+multiplicacao+"\nA subtracao = "+subtracao);


    }
}